integration-tests/test-calico/test_calico.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -eux
+
+# This job expect awscli and juju to be configured
+
+CHARM_CHANNEL=${CHARM_CHANNEL:-"candidate"}
+
+function cleanup {
+  juju destroy-controller aws-test-vpc --destroy-all-models -y || true
+  ./cleanup-vpc.sh
+}
+trap cleanup EXIT
+
+./integration-tests/test-calico/bootstrap-single-subnet-aws.sh
+
+# Deploy kubernetes with calico
+juju deploy cs:~containers/kubernetes-calico --channel ${CHARM_CHANNEL}
+#juju deploy cs:~cynerva/bundle/kubernetes-calico-9
```

Comment:
There's nothing special about this cynerva bundle, feel free to remove the comment :)

Reply:
Removed